### DISCUSSION TITLE
AIC dataset for pifpaf models

### DISCUSSION THREAD

**Discussion by chaowentao** [openpifpaf/openpifpaf#479] [Ideas]
Hi. I appreciate your great work.
I want to know whether there are scripts for training the AI challenger dataset for pifpaf models?

[a comment by an automated account is not shown]

**Comment by svenkreiss** [comment]
Hello @chaowentao , did you find anything? I am not aware of anyone who has made a plugin for AIC.

**Comment by chaowentao** [comment]
@svenkreiss Sorry, I also didn't find the plugin for AIC.
